llm_unlearn_ucl/unlearn_harm.py

```diff
@@ -707,7 +555,9 @@ def main(args) -> None:
     # Initialize logging
     if bool(args.wandb_log):
         wandb.init(
-            project=args.wandb_project_name, name=args.wandb_run_name, config=vars(args)
+            project=args.wandb_project_name,
```

Comment:
wandb modified in another pr regarding #117 , I think this pr should go before the pr about integrating accelerate and deepspeed

Reply:
Nothing changed here. It's just black not blacking properly.